templates/apiMultipartHelper.hbs

```diff
@@ -0,0 +1,6 @@
+import multer from 'multer'
+
+/**
+ * File upload handling for multipart operations in the {{className name}} API endpoint.
+ */
+export const upload = multer({storage: multer.memoryStorage()})
```

Comment:
Let's put the options onto their own line with a trailing comma... setting a good example.

```typescript
export const upload = multer({
    storage: multer.memoryStorage(),
})
```